core/Names.cc

```diff
@@ -72,6 +72,9 @@ string NameRef::showRaw(const GlobalState &gs) const {
                 case UniqueNameKind::Packager:
                     kind = "G";
                     break;
+                case UniqueNameKind::PackagerPrivate:
+                    kind = "V";
+                    break;
```

Comment:
Create a new name kind for `_Package_Private` to prevent unnecessary string comparisons when de-munging for LSP and related purposes.